### DISCUSSION TITLE
New color issues.

### DISCUSSION THREAD

**Discussion by dan4ik605743** [lowlighter/metrics#624] [Ideas]
GitHub has changed the color of the closed issue, change it in the github-metrics.
![image](https://user-images.githubusercontent.com/86075850/139165650-c7b34ede-61b5-49d8-bc18-724576b5ab57.png)
![image](https://user-images.githubusercontent.com/86075850/139165685-8578f2ef-2317-439c-8dfe-10e96986ee42.png)

**Comment by lowlighter** [comment]
(Tracked in #625)
